test/cbmc/proofs/Makefile-project-defines

```diff
@@ -13,6 +13,18 @@
 # Makefile.common.
 ################################################################
 
+# Name of this proof project, displayed in proof reports. For example,
+# "s2n" or "Amazon FreeRTOS". For projects with multiple proof roots,
+# this may be overridden on the command-line to Make, for example
+#
+# 	  make PROJECT_NAME="FreeRTOS MQTT" report
+#
+PROJECT_NAME = "FreeRTOS coreJobs"
```

Comment:
I think "coreJobs", though it has a nice right to it, isn't the right name.

Reply:
Changed to "Jobs library"